examples/android/local/app/src/androidTest/java/com/mongodb/realm/examples/kotlin/WritesTest.kt

```diff
@@ -86,4 +90,51 @@ class WritesTest : RealmTest() {
         }
         expectation.await()
     }
+
+    @Test
+    fun testCounter() {
+        val expectation = Expectation()
+        activity!!.runOnUiThread {
+            val config = RealmConfiguration.Builder()
+                .name("writes-test-counter")
+                .allowQueriesOnUiThread(true)
+                .allowWritesOnUiThread(true)
+                .inMemory()
+                .build()
+            val realm = Realm.getInstance(config)
+            realm.executeTransaction { r: Realm? ->
+                realm.createObject(
+                    HauntedHouseKt::class.java
+                )
+            }
+            // :replace-start: {
+            //    "terms": {
+            //       "HauntedHouseKt": "HauntedHouse"
+            //    }
+            // }
+            // :code-block-start: counter-increment-decrement
+            val house = realm.where(HauntedHouseKt::class.java)
+                .findFirst()!!
+            realm.executeTransaction {
+                Log.v("EXAMPLE", "Number of ghosts: ${house.ghosts.get()}") // 0
+                house.ghosts.increment(1)
+                Log.v("EXAMPLE", "Number of ghosts: ${house.ghosts.get()}") // 1
+                house.ghosts.increment(5)
+                Log.v("EXAMPLE", "Number of ghosts: ${house.ghosts.get()}") // 6
+                house.ghosts.decrement(2)
+                Log.v("EXAMPLE", "Number of ghosts: ${house.ghosts.get()}") // 2
```

Comment:
Shouldn't this be...
```suggestion
                Log.v("EXAMPLE", "Number of ghosts: ${house.ghosts.get()}") // 4
```